### ISSUE TITLE
[Feature Request] One-Clic install ?

### ISSUE THREAD

**Issue by neoluigi4123** [SociallyIneptWeeb/AICoverGen#57]
Can you make a executable that install AICoverGen ( for people having issue installing it localy (me) ) ?

**Comment by NazgulCoder** [comment]
I agree with you, I tried to deploy it locally from 2 different machines but it all returned me same error when trying to install Fairseq requirement.

I then tried on a Ubuntu machine and it worked flawlessly, however I am not able to compile it with AutoPy 2 Exe.
Would be great a docker container too

**Comment by laurentbenaroya** [comment]
I just made a docker container.
The files are [Dockerfile](https://gist.github.com/laurentbenaroya/af7d2378f6f211b9b75deae0f844cb1e#file-dockerfile), [requirements_docker.txt](https://gist.github.com/laurentbenaroya/af7d2378f6f211b9b75deae0f844cb1e#file-requirements_docker-txt)
Here are some explanations on how to use command-line code with this docker container. 
In short, the Dockerfile is based on a fairseq image. 
I've modified the requirements.txt file a little. 
It turns out that I installed something because Nvidia gpu was not supported in the container, 
but in fact the `--gpus all` option should suffice.

Here the explanation :
1) download the base docker container
``` docker pull alex2612/fairseq_audioaug_150823:latest```
2) go to top directory in AICoverGen
   in src/main.py, replace action=argparse.BooleanOptionalAction by **action='store_true'**
   it is a python version compatibility problem.

3) create a folder XXX in rvc_models (replace XXX by whatever you want). 
   Copy your rvc model in rvc_models/XXX, and unzip it there (if needed)

4) (?) Install the NVIDIA Container Toolkit (I'M NOT SURE THAT THIS IS NEEDED, PLEASE TRY FIRST WITHOUT THIS STEP)
   https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html

5) build docker image based on Dockerfile
``` docker build -t aicovergen . ```
6) run it !
``` docker run --gpus all -it --name aicovergencontainer aicovergen ```
then inside the docker :
``` 
python src/download_models.py
python src/main.py -i my_youtube_url -dir XXX -p 0 
```
7) finally in another terminal, copy the output files :
``` docker cp yyy:/app/song_output mysong ```
where yyy is the container ID or name (`docker ps` will give you the yyy)
